utils/gyb_syntax_support/DeclNodes.py

```diff
@@ -480,12 +480,14 @@
                    is_optional=True),
          ]),
 
+    # FIXME: technically misnamed; should be "ImportPathComponent"
     Node('AccessPathComponent', kind='Syntax',
          children=[
             Child('Name', kind='IdentifierToken'),
             Child('TrailingDot', kind='PeriodToken', is_optional=True),
          ]),
 
+    # FIXME: technically misnamed; should be "ImportPath"
     Node('AccessPath', kind='SyntaxCollection', element='AccessPathComponent'),
```

Comment:
Fixing this will change names in SwiftSyntax; I'm not sure that's quite worth it.